### ISSUE TITLE
A question about sharpness

### ISSUE THREAD

**Issue by twoentartian** [davda54/sam#96]
Hi, thanks for your work. I have a question about the concept of *sharpness*. More specifically, can I use the loss value obtained from `smooth_crossentropy` as  a metric of measuring the sharpness of the loss landscape near the model?

The code should looks like the following:

```python
enable_running_stats(model)
predictions = model(inputs)
loss = smooth_crossentropy(predictions, targets, smoothing=args.label_smoothing)
sharpness = loss.item()  # metric of measuring the sharpness of the loss landscape

# I guess I could remove the second forward-backward step because I do not need to optimize the model. 
```

However, when I check the `smooth_crossentropy`, I find it only operates on the labels, thus it shouldn't be a measure of loss landscape sharpness. Now I am a little bit lost. Is there a way to measure the sharpness?
